hikari/files.py

```diff
@@ -470,6 +480,40 @@ async def read(
         async with self.stream(executor=executor) as reader:
             return await reader.read()
 
+    async def save(
+        self,
+        path: typing.Optional[Pathish] = None,
+        *,
+        executor: typing.Optional[concurrent.futures.Executor] = None,
+        force: bool = False,
+    ) -> None:
+        """Save this resource to disk.
+
+        This writes the resource file in chunks, and so does not load
+        the entire resource into memory.
+
+        Parameters
+        ----------
+        path : typing.Optional[Pathish]
+            The path to save this resource to. If this is a string, the
+            path will be relative to the current working directory. If
+            `None`, the resource will be saved as `filename` in
+            the current working directory.
```

Comment:
Defaulting to saving as filename in the current working directory seems like odd behaviour, are you sure that's a good default behaviour to have?